README.md

```diff
@@ -42,7 +42,9 @@ represent the tiles.
 
 ## A1.1
 
-«replace this with your answer»
+S -> A
+S -> A S A
+A -> "r" | "g" | "b"
```

Comment:
# -3

Unfortunately this doesn't work, as "A S A" can match "r S b". You needed to constrain it with something like:

~~~
       S -> r | g | b
       S -> r  S  r
       S -> g  S  g
       S -> b  S  b
~~~